google-cloud-vision/lib/google/cloud/vision/project.rb

```diff
@@ -274,6 +275,76 @@ def annotate *images, faces: false, landmarks: false, logos: false,
           return annotations.first if annotations.count == 1
           annotations
         end
+
+        ##
+        # Run asynchronous image detection and annotation for a list of generic
+        # files, such as PDF files, which may contain multiple pages and
+        # multiple images per page. Progress and results can be retrieved
+        # through the google.longrunning.Operations interface.
+        # Operation.metadata contains OperationMetadata (metadata).
+        # Operation.response contains AsyncBatchAnnotateFilesResponse
+        # (results).
+        #
+        # @param requests
+        #   [Array<Google::Cloud::Vision::V1::AsyncAnnotateFileRequest | Hash>]
+        #   Individual async file annotation requests for this batch.
+        #   A hash of the same form as
+        # `Google::Cloud::Vision::V1::AsyncAnnotateFileRequest` can also be
+        #   provided.
+        # @param options [Google::Gax::CallOptions]
+        #   Overrides the default settings for this call, e.g, timeout,
+        #   retries, etc.
+        # @return [Google::Gax::Operation]
+        # @raise [Google::Gax::GaxError] if the RPC is aborted.
+        # example
+        #   require "google/cloud/vision"
```

Comment:
Oh and since all this is no longer in an `@example` block, it probably needs to be indented two more spaces (inside the comment) so markdown will treat it as code. (You can `bundle exec rake yardoc` and double check the resulting html to be sure.)